docs/developingWorkflows/developing.rst

```diff
@@ -24,7 +24,7 @@ workflow::
 
     if __name__=="__main__":
         options = Job.Runner.getDefaultOptions("./toilWorkflowRun")
-        options.logLevel = "INFO"
+        options.logLevel = "DEBUG"
```

Comment:
I think the docs should run with INFO by default. What was your thoughts when you moved it down?

Probably this line should be deleted all together and logLevel should just assume the default value.